mypy/test/testpythoneval.py

```diff
@@ -71,6 +71,9 @@ def test_python_evaluation(testcase: DataDrivenTestCase, cache_dir: str) -> None
                     return
         mypy_cmdline.extend(additional_flags)
 
+    if "--no-force-union-syntax" not in mypy_cmdline:
+        mypy_cmdline.append("--force-union-syntax")
```

Comment:
Adding this here temporarily. Without it the output of a lot of pythoneval test would change now that these are run with `3.10` by default.

I plan to update the test outputs to use the PEP 604 union syntax in separate PRs later.